modules/storage-persistent-storage-aws-maximum-volumes.adoc

```diff
@@ -7,13 +7,17 @@
 
 By default, {product-title} supports a maximum of 39 EBS volumes attached to one
 node. This limit is consistent with the
-link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/volume_limits.html#linux-specific-volume-limits[AWS
-volume limits].
+link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/volume_limits.html#linux-specific-volume-limits[AWS volume limits].
+
+[IMPORTANT]
+====
+As a cluster administrator, you must use either in-tree volumes or Container Storage Interface (CSI) volumes, but never both at the same time. The maximum attached EBS volume number is counted separately for in-tree and CSI volumes.
+====
 
 {product-title} can be configured to have a higher limit by setting the
 environment variable `KUBE_MAX_PD_VOLS`. However, AWS requires a particular
 naming scheme
 (link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/device_naming.html[AWS
 Device Naming]) for attached devices, which only supports a maximum of 52
-volumes. This limits the number of volumes that can be attached to a node via
-{product-title} to 52.
+volumes. The number of volumes that can be attached to a node using
+{product-title} is therefore limited to 52.
```

Comment:
From this doc: https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/volume_limits.html#linux-specific-volume-limits, looks the limit is not 52.  Tested with m5 VM, its limit is not 52.

Reply:
Based on Qin's next comment, I removed the comments about using`KUBE_MAX_PD_VOLS`.